<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormFormulaToMass</class>
 <widget class="QWidget" name="FormFormulaToMass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="labelInputFormula">
       <property name="text">
        <string>Input formula:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="LabeledText" name="textInputFormula">
       <property name="placeholderText">
        <string notr="true">C6H12O6</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonGetMass">
       <property name="toolTip">
        <string>Calculate the molecular weight of input formula</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/icons/return.png</normaloff>:/icons/return.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonModification">
       <property name="toolTip">
        <string>Set modification</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/icons/view-more-horizontal.png</normaloff>:/icons/view-more-horizontal.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="FrameFormulaModification" name="frameModification">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="labelResultMonoMass">
       <property name="text">
        <string>Monoisotopic mass:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="textResultMonoMass">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="labelAverageMass">
       <property name="text">
        <string>Average mass:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="textResultMass">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Emperical formula:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="textResultFormula"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>41</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FrameFormulaModification</class>
   <extends>QFrame</extends>
   <header>widget/frameformulamodification.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LabeledText</class>
   <extends>QLineEdit</extends>
   <header>widget/labeledtext.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
